platform/solr/solr-dependencies/pom.xml

```diff
@@ -27,7 +27,6 @@
         <info.ganglia.gmetric4j.version>1.0.7</info.ganglia.gmetric4j.version>
         <org.slf4j.version>1.7.7</org.slf4j.version>
         <log4j.version>1.2.17</log4j.version>
-        <jts.version>1.13</jts.version>
     </properties>
     <dependencies>
```

Comment:
❗️ Were there no other dependency changes?  These are all of the dependencies and their versions required by Solr 6.6.0 which I got from `solr-6.6.0/server/solr-webapp/webapp/WEB-INF/lib`.  I think JTS should be the only addition.  For example, I see janino in the list below but I do not see it in the list of jars that will be shaded in.

```
antlr4-runtime-4.5.1-1.jar
asm-5.1.jar
asm-commons-5.1.jar
avatica-core-1.9.0.jar
caffeine-2.4.0.jar
calcite-core-1.11.0.jar
calcite-linq4j-1.11.0.jar
commons-cli-1.2.jar
commons-codec-1.10.jar
commons-collections-3.2.2.jar
commons-compiler-2.7.6.jar
commons-configuration-1.6.jar
commons-exec-1.3.jar
commons-fileupload-1.3.2.jar
commons-io-2.5.jar
commons-lang-2.6.jar
commons-math3-3.4.1.jar
curator-client-2.8.0.jar
curator-framework-2.8.0.jar
curator-recipes-2.8.0.jar
dom4j-1.6.1.jar
eigenbase-properties-1.1.5.jar
guava-14.0.1.jar
hadoop-annotations-2.7.2.jar
hadoop-auth-2.7.2.jar
hadoop-common-2.7.2.jar
hadoop-hdfs-2.7.2.jar
hppc-0.7.1.jar
htrace-core-3.2.0-incubating.jar
httpclient-4.4.1.jar
httpcore-4.4.1.jar
httpmime-4.4.1.jar
jackson-annotations-2.5.4.jar
jackson-core-2.5.4.jar
jackson-core-asl-1.9.13.jar
jackson-databind-2.5.4.jar
jackson-dataformat-smile-2.5.4.jar
jackson-mapper-asl-1.9.13.jar
janino-2.7.6.jar
joda-time-2.2.jar
lucene-analyzers-common-6.6.0.jar
lucene-analyzers-kuromoji-6.6.0.jar
lucene-analyzers-phonetic-6.6.0.jar
lucene-backward-codecs-6.6.0.jar
lucene-classification-6.6.0.jar
lucene-codecs-6.6.0.jar
lucene-core-6.6.0.jar
lucene-expressions-6.6.0.jar
lucene-grouping-6.6.0.jar
lucene-highlighter-6.6.0.jar
lucene-join-6.6.0.jar
lucene-memory-6.6.0.jar
lucene-misc-6.6.0.jar
lucene-queries-6.6.0.jar
lucene-queryparser-6.6.0.jar
lucene-sandbox-6.6.0.jar
lucene-spatial-extras-6.6.0.jar
lucene-suggest-6.6.0.jar
noggit-0.6.jar
org.restlet-2.3.0.jar
org.restlet.ext.servlet-2.3.0.jar
protobuf-java-3.1.0.jar
solr-core-6.6.0.jar
solr-solrj-6.6.0.jar
spatial4j-0.6.jar
stax2-api-3.1.4.jar
t-digest-3.1.jar
woodstox-core-asl-4.4.1.jar
zookeeper-3.4.10.jar
```